packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -128,15 +128,18 @@ const _DropdownOverlay = ({
   return (
     <BaseBox position="relative">
       <StyledDropdownOverlay
-        width={width}
+        width={isMenu ? undefined : width}
+        minWidth="240px"
+        // in SelectInput, we don't want to set maxWidth because it takes width according to the trigger
+        maxWidth={isMenu ? '400px' : undefined}
+        left={isMenu ? 'spacing.0' : undefined}
```

Comment:
There is one use case where keeping `left='0'` might overflow on the right.
Can we add styled props support for aligning the position of the dropdown overlay?

<img width="249" alt="image" src="https://github.com/razorpay/blade/assets/48921037/6c7df685-b7f4-4e20-8a53-e40891069e44">

Reply:
Started discussion for styled-props internally. Will update you on that. we can maybe merge minWidth and maxWidth changes first and will keep discussing styled-props parallely meanwhile?